src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,13 @@
+import jwt from 'jsonwebtoken';
+
+export default (req, res, next) => {
+  const token = req.header('auth-token');
+  if (!token) return res.status(401).send({ error: 'Acces Denied' });
+  try {
+    const verified = jwt.verify(token, process.env.JWT_SECRET);
+    req.user = verified;
+    next();
+  } catch (err) {
+    res.status(400).send({ error: 'Invalid Token' });
+  }
```

Comment:
This feature was already implemented in another file. Check this [file](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/87a3307d2ce0a9fcba8f9966f9735e0d6488d501/src/middlewares/auth.middleware.js#L5) and see if you can use it.